README.md

```diff
@@ -3,6 +3,9 @@ A place to organize style guides, best practices, tools, and techniques for Stan
 
  - [Best Practices](/best-practices)
    - [Documentation](/best-practices/documentation)
+   - [How To - Readme](/best-practices/howto_readme.md)
+   - [Licensing](/best-practices/licensing.md)
+   - [Version Control](/best-practices/version_control.md)
```

Comment:
Sounds like a good idea.  Agree with Jack that it'll become clear whether we need it or not as the playbook gets larger. I'll file a meta issue on it so we don't lose track of the idea.